src/celengine/nebula.cpp

```diff
@@ -62,6 +88,10 @@ Nebula::getObjType() const
 bool
 Nebula::load(const util::AssociativeArray* params, const fs::path& resPath, std::string_view name)
 {
+    
+    if (const auto* typeName = params->getString("Type"); typeName != nullptr)
```

Comment:
```suggestion
    if (const std::string* typeName = params->getString("Type"); typeName != nullptr)
```
The next line uses `std::string`, so let's keep the same style.